lib/marqeta/card.rb

```diff
@@ -0,0 +1,28 @@
+module Marqeta
+  class Card < ApiObject
+    ACTIVE_STATE = 'ACTIVE'.freeze
+
+    def self.endpoint
+      'cards'
+    end
+
+    def self.from_pan(pan)
```

Comment:
Is pan a marqeta term?

Reply:
It's a credit card term: https://en.wikipedia.org/wiki/PAN_truncation